### DISCUSSION TITLE
GET listing of datasets for a study

### DISCUSSION THREAD

**Discussion by swhume** [cdisc-org/DataExchange-DatasetJson-API#4] [User Stories]
As a sponsor, I want to get a listing of all available datasets for a specified study so that I can retrieve each dataset individually

**Comment by chowsanthony** [comment]
In the response payload, would it be beneficial to differentiate reference data from clinical data? Is it more appropriate to have a separate endpoint for that purpose?

**Reply by swhume** [reply]
We could have separate endpoints. I assumed we'd have 1 endpoint for datasets that would return both clinicalData or referenceData to keep the API simple, but it may be that having different endpoints makes processing a bit easier. I'm interested to hear what others think on this one.

**Reply by CellCS** [reply]
One optional parameter could be added in this endpoint. Like "studies" , "studies/{clinicaldata}", "studies/{referencedata}". 
If considering that total number of studies may have large number, could also add "/{limit}" parameter or compressed parameter like "/{zip}" to indicate compressed format.  There are more possibilities. Please add change request, so will add into implementation.
